src/zenml/cli/stack.py

```diff
@@ -32,11 +34,13 @@ def stack() -> None:
 @click.option("-m", "--metadata-store", type=str, required=True)
 @click.option("-a", "--artifact-store", type=str, required=True)
 @click.option("-o", "--orchestrator", type=str, required=True)
+@click.option("-c", "--container_registry", type=str, required=False)
```

Comment:
We shouldn't forget to add this option in gitbook (https://app.gitbook.com/o/-MCl1Hlw9oU4xibZ-ymf/s/-MCl0MaN3pJfPa2vq2Rb/guides/deploy_to_production)